### ISSUE TITLE
Console for my project now shows multiple warnings for running in development mode 

### ISSUE THREAD

**Issue by kahunacoder** [gwenaelp/vfg-field-array#7]
When I added ```vfg-field-array``` and ```vfg-field-object``` to my vue project I get warning for each of these libraries that I'm running in development mode. The warnings show the files are ```vfg-field-array.esm.js:9392``` and ```vfg-field-object.esm.js:9128```. 

This happens when I'm running my project in development mode. Seems like it should only happen if I'm running one of these libraries in development mode, not when I'm running my project in development mode.

I just thought you should know.  I like these libraries and plan on using them in multiple projects.

**Comment by gwenaelp** [comment]
Yes there is a problem with the build system. This also impacts the weight of the libraries. I will refactor the project architecture this month, to leave everything way more simple for people using them.

Nevertheless, it will probably be less straightforward to use for people that does not have webpack and vue loader (in a nutshell, people that are not able to load .vue files in their project). But nowadays everybody tends to have them so I think there will be no problem at all.

Thanks for the feedback!

If you need help of any kind, I'll be glad to have more feedback on those libraries ! =D

I leave this issue opened and I will close it when I will have refactored vfg-field-array and vfg-field-object.
